dj/api/graphql/column.py

```diff
@@ -1,38 +0,0 @@
-"""
```

Comment:
These don't get added back right? Is the idea that we can add the graphql API later, after the existing rest API is more settled?

Reply:
Yeah we needed to thoroughly comb through the graphql API anyway so it seemed much easier to add this back and confirm parity when we're locked into an openapi spec for an entire major version (cc: @CircArgs).

Reply:
Makes sense!